### PR TITLE
Update istio-latest to 1.8.1

### PR DESCRIPTION
**Pull request by arturenault**
# Changes

- Update istio-latest to 1.8.1.

### CODE REVIEW

File: third_party/istio-latest/istio-ci-mesh.yaml [hunk 1]
```diff
@@ -43,8 +43,6 @@ spec:
   addonComponents:
     pilot:
       enabled: true
-    prometheus:
```
Comment (ZhiminXiang) [inline third_party/istio-latest/istio-ci-mesh.yaml:46]: out of curiosity, why does this need to be deleted?

Comment (arturenault) [reply third_party/istio-latest/istio-ci-mesh.yaml:46]: details here: https://github.com/knative/serving/issues/10221

**Pull request merged**